app/assets/javascripts/components/course_queue.js.jsx

```diff
@@ -177,9 +177,9 @@ var CourseQueue = React.createClass({
           this.broadcastInstructorMessage(data);
         } else if (data.action === 'bump'
                    && data.requester_id === this.props.currentUserId) {
-           this.notify(data.bump_by.name + ' is looking for you!', true, {
```

Comment:
Think it'd be useful to keep the instructor's name in there? "Matt says: I'm looking for you" 🤷‍♂️

Reply:
Fixed!